test/HiGlassComponentTest.js

```diff
@@ -1802,57 +1801,58 @@ describe('Simple HiGlassComponent', () => {
 
 
     it('Moves the brush on one of the views', (done) => {
-      const heatmapTrack = getTrackObjectFromHGC(hgc.instance(), 'aa', 'heatmap1');
+      const track = getTrackObjectFromHGC(hgc.instance(), 'aa', 'heatmap1');
 
-      const domain1 = heatmapTrack.limitedValueScale.domain();
+      const domain1 = track.limitedValueScale.domain();
 
 
-      heatmapTrack.gColorscaleBrush.call(heatmapTrack.scaleBrush.move,
+      track.gColorscaleBrush.call(track.scaleBrush.move,
         [0, 100]);
 
-      const domain2 = heatmapTrack.limitedValueScale.domain();
+      const domain2 = track.limitedValueScale.domain();
 
       // we don't expect the other view to change
       expect(domain1[0]).not.toEqual(domain2[0]);
 
-
       // console.log('domain1:', domain1);
       // console.log('domain2:', domain2);
 
-
       done();
     });
 
     it('locks the scales and recenters the page', (done) => {
       hgc.instance().handleValueScaleLocked('aa', 'heatmap1', 'view2', 'heatmap2');
-      const track1 = getTrackObjectFromHGC(hgc.instance(), 'aa', 'heatmap1');
-      const track2 = getTrackObjectFromHGC(hgc.instance(), 'view2', 'heatmap2');
+      getTrackObjectFromHGC(hgc.instance(), 'aa', 'heatmap1');
+      getTrackObjectFromHGC(hgc.instance(), 'view2', 'heatmap2');
```

Comment:
What is the point of the two functions above if the return value is not used? I suspect they can be removed?

Reply:
See Pete's comment below: I did not want to mistakenly loosen the tests, so even if the value is unused, I've generally left the calls in place. If you are confident that in a particular case there are no side effects we're trying to test, glad to get rid of it, but not a call I felt comfortable making.